docs/NLUGitHubActions.md

```diff
@@ -0,0 +1,85 @@
+## GitHub Actions workflow using NLU.DevOps cli tool
+
+This documents walk through building a CI pipeline for LUIS using GitHub Actions and NLU.DevOps.
+The workflow will be the following: 
+1. create, train, publish the LUIS model using sample utterances
+2. send a test set to created LUIS model
+3. evaluate model by comparing results received from LUIS with expected values
+4. delete the model from the portal
+
+Let's name this pipeline CINLU and run it on a pull request to GitHub repo.
+
+```
+name: CINLU
+
+on: [pull_request]
+```
+
+1. Let's install NLU.DevOps cli tool on GitHub agent.
+
+```
+    - name: Install dotnet-nlu
+      run: dotnet tool install -g dotnet-nlu
+```
+On the ubuntu agent, you need to prepend a directory to the system PATH variable for all subsequent actions in the current job to make sure that cli tool works. More about this command [here](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/development-tools-for-github-actions#add-a-system-path-add-path).
+
+```
+    - name: Path
+      run: echo "::add-path::$HOME/.dotnet/tools"
+```
+We use [utterances.json](../models/utterances.json) for training. You can replace this file with another file that consists of intents, utterances, entities that you need for your own model.
```

Comment:
>utterances [](start = 35, length = 10)

I think you wanted utterancesTest.json here.
do you need to provide a testing file for utterances? the examples you are using are already in the repo can we use them instead ?